src/styles/styles.js

```diff
@@ -1514,7 +1513,13 @@ const styles = {
     },
 
     overlayStyles: (current) => ({
-        ...StyleSheet.absoluteFillObject,
+        position: 'fixed',
```

Comment:
Yeah, but the overlay is visible only for wide view and that's something we don't support yet on iOS and Android.

Reply:
ios safari with ipad on landscape?

Reply:
By iOS and Android, I meant the native apps. It will work on safari/chrome mobile. 

We could think about alternative solutions but supporting three pane view in native apps requires more changes anyway. I am not sure if it's worth doing now if the code can change.

Reply:
@rushatgabhane I think it does not work at all in native ipad now so we should not spend time on it now, but I agree that making it work in ipad landscape should be on our list soon